add_balancer.yml

```diff
@@ -40,10 +40,10 @@
       tags: always
 
     # Stop, if no nodes found with new_node variable
-    - name: "Pre-Check error. No nodes found with new_node set to true"
+    - name: Pre-Check error. No nodes found with new_node set to true
       run_once: true # noqa run-once
       fail:
-        msg: "Please specify the new_node=true variable for the new balancer server to add it to the existing cluster."
+        msg: Please specify the new_node=true variable for the new balancer server to add it to the existing cluster.
```

Comment:
as a best practice, I would recommend using double quotes for greater clarity and predictability.